packages/hui-pages/src/routes/TextDocs/index.js

```diff
@@ -57,17 +56,17 @@ const TextDocs = () => (
         <Text jumbo>jumbo</Text>
       </Col>
       <Col xs={3}>
-        <Text thinnest>thinnest</Text>
+        <Text larger thinnest>thinnest</Text>
```

Comment:
Did thinnest and thinner need to be larger than the other weighted examples?

Reply:
Yep - you can't see the thinner and thinnest at 'normal', needs to be 'larger'